components/brave_wallet/browser/bitcoin/bitcoin_wallet_service.cc

```diff
@@ -199,47 +197,52 @@ void BitcoinWalletService::Bind(
   receivers_.Add(this, std::move(receiver));
 }
 
-void BitcoinWalletService::GetBitcoinAccountInfo(
-    const std::string& network_id,
-    mojom::KeyringId keyring_id,
-    uint32_t account_index,
-    GetBitcoinAccountInfoCallback callback) {
-  if (!IsValidBitcoinNetworkKeyringPair(network_id, keyring_id)) {
-    NOTREACHED();
-    std::move(callback).Run(nullptr);
+void BitcoinWalletService::GetBalance(const std::string& network_id,
+                                      mojom::AccountIdPtr account_id,
+                                      GetBalanceCallback callback) {
+  auto account_info =
+      GetBitcoinAccountInfoSync(network_id, std::move(account_id));
+  if (!account_info) {
+    std::move(callback).Run("0", "Couldn't get balance");
```

Comment:
we need l10n string for this error

Reply:
I've intentially left localization in this file to be done when whole feature is almost ready.

Reply:
please also open an issue for it so we won't forget.